### DISCUSSION TITLE
WYSIWYG revision tracking

### DISCUSSION THREAD

**Discussion by DustinBracy** [directus/directus#18529] [Feature Requests]
I love the revision history feature, and I really wish I could use it on WYSIWYG blocks.  It works, but changing one word results in the entire text body being red as 'changed', and the entire body in green as 'new'.  Not super helpful for a long document (we're using this for internal procedures, which are typically a page or two long of formatted text).   Is there something I can do to only show me the highlighted word change?   For instance, if I only change one word in a paragraph, I'd really like to just see the one word that was changed.  Markdown fields support this much better than WYSIWYG fields, but teaching my end users how to use markdown is probably not realistic.  Is this related to the HTML styling being stored as a single, long text field?  Does anyone have a workaround?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
